src/components-v1/MyProfile.vue

```diff
@@ -134,9 +138,21 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-.wrapper {
-  background-color: #f8f8f8;
-  min-height: 400px;
-  padding: 8px;
+@import '~bh/src/assets/scss/colors.scss';
+@import '~bh/src/assets/scss/variables.scss';
+
+
+.my-profile {
+  &__header {
+    background-color: $color-primary;
+  }
+
+  &__repositories {
+    margin: $size-small 8rem;
```

Comment:
Don't use margin in this case, make delimiter like in repository card:
https://github.com/Ilhasoft/bothub-webapp/blob/develop/src/components/repository/RepositoryViewBase/RepositoryViewBase.vue#L338